guides/choosing_mono_or_multi_repo.md

```diff
@@ -0,0 +1,96 @@
+# Choosing between a mono repo and a multi repo
+
+A mono repo is a single repository that contains all the code for multiple projects. A multi repo is a collection of repositories, each containing the code for a single project. Both approaches have their own advantages and disadvantages. This guide will help you decide which approach is best for your project.
+
+Developers should proactively choose between a mono repo and a multi repo based on the project requirements. The decision should be made at the beginning of the project, as it can be difficult to switch between the two later on.
```

Comment:
How difficult is it? Is it more difficult to switch one way or the other? Would that help to provide a steer on a preferred starting point?

It can sometimes be pretty much impossible to judge this kind of thing ahead of time, so my preference would be to provide guidance that makes it as easy as possible to change tack down the road if necessary.

Reply:
Guidance would be helpful. For example, keeping the commit history would be important if a project is switching.

Reply:
I've added a line about what influences the complexity.  There's no one answer to how complicated it is as it's very subjective on tech, CI/CD and history requirements.  But what covers all of them is time, the longer a repo has existed, typically the harder it is to unpick.